scripts/scheming_migration/index.js

```diff
@@ -334,16 +365,16 @@ async function main() {
 					+ "($10, $1, 'publish_state', $5, $6, 'active')";
 
 			let extrasUpdateValues = [
-				packageObj['id'], 
-				JSON.stringify(packageExtras['contacts']),
-				JSON.stringify(packageExtras['dates']),
-				JSON.stringify(packageExtras['more_info']),
-				packageExtras['edc_state'],
-				packageObj['revision_id'],
-				uuidv4(),
-				uuidv4(),
-				uuidv4(),
-				uuidv4()
+				packageObj['id'], //1
```

Comment:
Are these comments?  Did you mean to add them?

Reply:
yes I meant to add them, I initially tried to do the change as part of that query and I found it impossible to read without the number there that corresponds to the number it is in the query above.

Reply:
Thanks - that makes sense.